firstvoices/backend/tests/test_models/test_song_model.py

```diff
@@ -9,3 +11,15 @@ class TestSongModel(RelatedVideoLinksValidationMixin):
 
     def create_instance_with_related_video_links(self, site, related_video_links):
         return SongFactory.create(site=site, related_video_links=related_video_links)
+
+    @pytest.mark.django_db
+    def test_html_cleaning_fields(self):
```

Comment:
These tests don't need to go on the models-- all the logic is in the field, so please just add one test file with tests for the field.

Reply:
When you do that, can you please add a test case where we save a non-html string that has some html-related characters in it? I am thinking about the widget settings using the HTML fi1eld for every value. So e.g., the text could be "Why is 1 < 2 ?" or "The French person said << quoi ?? >>"